packages/design-system/src/styles/components/_Badge.scss

```diff
@@ -2,6 +2,26 @@
 
 $badge-font-size-big: $base-font-size;
 
+@mixin badge-high-contrast-mode-remove-color() {
+  /* stylelint-disable scss/media-feature-value-dollar-variable */
+  @media (-ms-high-contrast: active), (forced-colors: active) {
+    background-color: windowText;
+    color: window;
+    -ms-high-contrast-adjust: none;
+  }
+  /* stylelint-enable scss/media-feature-value-dollar-variable */
+}
+
```

Comment:
Wondering if any of these mixins are generic enough to be applied to our WHCM fixes?